src/main/java/org/jitsi/videobridge/BitrateController.java

```diff
@@ -685,12 +685,24 @@ private EndpointBitrateAllocation(
 
             // Initialize rates.
             rates = new long[encodings.length];
+            int optimal180Idx = 0;
```

Comment:
maybe this should be `optimalThumbnailIndex` and instead of hardcoding `180` below we make a `THUMBNAIL_MAX_RESOLUTION` field or something?